### PR TITLE
Update all packages

### PR DESCRIPTION
**Pull request by tmo207**
# Update all packages

Updated all packages to the newest version.

- [x] I have added or updated unit tests for this change.
- [x] I have updated the [changelog](https://github.com/rametta/rapini/blob/main/changelog.md) with info about the changes in this PR

### CODE REVIEW

File: spec/common/util.spec.ts [hunk 1]
```diff
@@ -103,12 +103,12 @@ describe("nodeId", () => {
...
-    ${bool}    | ${"133"}
-    ${any}     | ${"130"}
-    ${union}   | ${"147|150|MyTypeName"}
-    ${arr}     | ${"Array<(147|150|MyTypeName)>"}
+    ${numeric} | ${"150"}
+    ${string}  | ${"154"}
+    ${bool}    | ${"136"}
+    ${any}     | ${"133"}
+    ${union}   | ${"150|154|MyTypeName"}
+    ${arr}     | ${"Array<(150|154|MyTypeName)>"}
```
Comment (rametta) [inline spec/common/util.spec.ts:111]: Would you be able to revert these changes please? Unless there was a specific reason for changing the numbers?

**Comment by rametta** [comment]
@tmo207 thanks for the PR! just 1 small comment about reverting the test file changes

File: spec/common/util.spec.ts (hunk 1 above)
Comment (tmo207) [reply spec/common/util.spec.ts:111]: The values in the TS definitions changed with the TS update, so this test has to be adapted in order to pass.

**Comment by rametta** [comment]
Thanks @tmo207, version 3.4.0 has been deployed to npm and ready to use
